### PR TITLE
Logical operations on Props.

### PR DESCRIPTION
**Pull request by randair**
(Oops forgot the name of the branch was still "rewrites").
New constants included in constants/proputils.ml:
* `or_introl`
* `or_intro`
* `conj`

### CODE REVIEW

File: src/coq/constants/proputils.ml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+let logical_or : types =
+  mkInd (MutInd.make1 (KerName.make2 coq_init_logic (Label.make "or")), 0)
+  
+(* left constructor of \/ *)
+let or_introl : types =
+  mkConstruct (fst (destInd logical_or), 1)
+
+(* right constructor of \/ *)
+let or_intror : types = 
+  mkConstruct (fst (destInd or), 2)
```
Comment (tlringer) [inline src/coq/constants/proputils.ml]: `logical_or`?

**Review by tlringer — CHANGES_REQUESTED** [review]
Just one thing looks like a mistake, everything else looks good

File: src/coq/constants/proputils.ml (hunk 1 above)
Comment (randair) [reply src/coq/constants/proputils.ml]: Yes that's right. Thank you I forgot to change it when I realized "or" is a reserved keyword.

Comment (tlringer) [reply src/coq/constants/proputils.ml]: Let me know when you've updated this so I can approve it

**Comment by randair** [comment]
[build.sh](https://github.com/uwplse/coq-plugin-lib/files/4399838/output.txt)

**Review by tlringer — APPROVED** [review]

**Pull request merged**